pset-functions.js

```diff
@@ -135,6 +160,22 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 3:')
 
 // Add your code below this line
+  const scrabbleLetters =new Map([
```

Comment:
@jimernest90 nice approach, I don't use `Map` much, but I like how you used this to build an object of letters and scores as key value pairs